cmd/icingadb/main.go

```diff
@@ -172,7 +172,7 @@ func run() int {
 		for hactx.Err() == nil {
 			select {
 			case takeoverReason := <-ha.Takeover():
-				logger.Infow("Taking over", zap.String("reason", takeoverReason))
+				logger.WithOptions(logs.ForceLog()).Infow("Taking over", zap.String("reason", takeoverReason))
```

Comment:
On second thought, I would rather create both messages with the warning severity. If someone wants to run Icinga DB with error as the default log level, they may really not be interested in anything but the worst.

So I would use the force log features only for the startup message.

What do you think, @lippserd?

Reply:
Okay :)